src/parser/h1_parser.py

```
@@ -1,259 +1,225 @@
+
+    def parse(self, data: bytes) -> Optional[Union[Dict[str, str], List[Dict[str, str]]]]:
         """
-        H1レコードをパースしてフィールド辞書を返す
-
-        Args:
-            data: パース対象のバイトデータ
+        H1レコードをパースする。
 
-        Returns:
-            フィールド名をキーとした辞書、エラー時はNone
+        28,955バイトのフルストラクトの場合: List[Dict] を返す（各組合せ1行）。
```

Ruff (0.14.14)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)